### DISCUSSION TITLE
天翼云盘的挂在报错

### DISCUSSION THREAD

**Discussion by ChenC2022** [AlistGo/alist#4450] [General]
自5月的某日开始，Alist 挂在的天翼云盘就无法正常使用了。之前一切正常，也未改过任何设置。
有其他用天翼的朋友出现同类问题吗？

error ：
failed get objs: failed to list objs: get page: <!DOCTYPE html><html><head><script type="text/javascript">var _uxt = _uxt || []; var _ux21st = (new Date()).getTime(); (function() { var ma = document.createElement('script'); ma.type = 'text/javascript'; ma.async = 'async'; ma.src = ('https:' == document.location.protocol ? 'https://ux.21cn.com' : 'http://ux.21cn.com') + '/api/htmlReportRest/getJs.js?pid=9A46A2CE...................................................

**Comment by anwen-anyi** [comment]
换`天翼云盘客户端`驱动

**Reply by Benson80** [reply]
换天翼云盘客户端提示Get "https://open.e.189.cn/api/logbox/oauth2/unifyAccountLogin.do?appId=8025431004&version=v1.1&clientType=10020&format=redirect¶s=301654E627E67E832ACC35E30AE3625A9B810A6513D536EC4755C40ADFD1DC9F0474060C960A6EB6866EC96FB3A8D811CB0F0CAADA03842BE142CFCFC7E8E63853223D1ED6F2253A0DBD6987B0CE1246F991B131F733848F71CACAC3C48D567C796ECABCE7A304A38A480B4E443A19B8&sign=C27972A03DC78A3B9DBF93BA1893FE8F19402ACB": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Reply by Benson80** [reply]
现在天翼云盘不能用了吗？

**Comment by mzsy576285** [comment]
天翼云盘客户端 好像也挂不了了，不知道怎么回事，报错 Request failed with status code 500，那个客户端 验证码该如何填写？

**Comment by Troray** [comment]
`v3.26.0`
`天翼云盘客户端`
![](https://i.imgur.com/Y66tY0K.png)

**Comment by captain-lu-A320** [comment]
2024年，天翼云盘目前搞不明白，cookie也不行，客户端也不行

**Reply by kokikoku** [reply]
点击编辑，把不使用OCR 按钮打开。
就可以看到家庭云里的内容。
刚上手。
